profiles/common/modules/custom/tmgmt_poetry/tmgmt_poetry_mock/src/Mock/PoetryMock.php

```diff
@@ -268,11 +268,10 @@ private static function translateRequestContent($content, $language) {
     $xml_content = simplexml_load_string($decoded_content);
     // Add language prefix to the title and body first paragraph.
     $title = (string) $xml_content->body->div->div[0];
-    $body = (string) $xml_content->body->div->div[1];
     // Overwriting title with language prefix.
     $xml_content->body->div->div[0] = "[$language] " . $title;
-    // Overwriting body with language prefix.
-    $xml_content->body->div->div[1] = '<p>' . "[$language] " . $body . '</p>';
```

Comment:
By this we are losing body content. But indeed it is not something important and can cause other issues.